Tasks/PWGHF/taskJpsi.cxx

```diff
@@ -47,6 +47,10 @@ struct TaskJpsi {
   Configurable<bool> d_modeJpsiToMuMu{"d_modeJpsiToMuMu", false, "Perform Jpsi to mu+mu- analysis"};
   Configurable<double> cutYCandMax{"cutYCandMax", -1., "max. cand. rapidity"};
   Configurable<std::vector<double>> bins{"pTBins", std::vector<double>{hf_cuts_jpsi_toee::pTBins_v}, "pT bin limits"};
+  Configurable<bool> selectedTofPid{"selectedTofPid", false, "select tof PID for Jpsi"};
+  Configurable<bool> selectedRichPid{"selectedRichPid", false, "select rich PID for Jpsi"};
+  Configurable<bool> selectedCombPid{"selectedCombPid", false, "select tof and rich PID for Jpsi"};
+  Configurable<bool> selectedMid{"selectedMid", false, "select MID PID for Jpsi to mu+mu-"};
```

Comment:
Please unify the configurable names and fix the capitalisation of detector names in the helper strings.